packages/cozy-harvest-lib/src/components/AccountForm/index.jsx

```diff
@@ -35,8 +35,10 @@ export class AccountForm extends PureComponent {
   }
 
   handleKeyUp(event, { dirty, form, initialValues, valid, values }) {
+    const key = event.code || event.key
```

Comment:
Is it because the event is differnt between preact and react ?

If so i think this can be more explicit by doing

```
const keyFromPreactOrReactEvent = event => event.code || event.key

const key = keyFromPreactOrReactEvent(event)
```

This way we do not need a comment. It could be done in a separate compat.js file also. I am a bit surprised than preact and react do not have the same structure for events :/

Reply:
Did you try `event.persist()`? React uses synthetic event but I don't know for preact

Reply:
Actually `event.code` [seems to be a standard](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code), but [not yet implemented in React](https://github.com/facebook/react/issues/14102). I will do a few more test to see what's the best property to rely on.

Reply:
`event.key` is working in both environment. Let's keep it and only it. 👌